docs/sources/flow/tasks/migrate/from-otelcol.md

```diff
@@ -0,0 +1,276 @@
+      exporters: [otlp]
+    traces:
+      receivers: [otlp]
+      processors: [memory_limiter]
+      exporters: [otlp]
+```
+
+The convert command takes the YAML file as input and outputs a [River][] file.
+
+{{< code >}}
+
+```static-binary
+AGENT_MODE=flow grafana-agent convert --source-format=otelcol --output=<OUTPUT_CONFIG_PATH> <INPUT_CONFIG_PATH>
+```
+
+```flow-binary
+grafana-agent-flow convert --source-format=otelcol --output=<OUTPUT_CONFIG_PATH> <INPUT_CONFIG_PATH>
+```
+
+{{< /code >}}
+
+Replace the following:
+
+- _`<INPUT_CONFIG_PATH>`_: The full path to the OpenTelemetry Collector configuration.
+- _`<OUTPUT_CONFIG_PATH>`_: The full path to output the {{< param "PRODUCT_NAME" >}} configuration.
+
+The new {{< param "PRODUCT_NAME" >}} configuration file looks like this:
+
+```river
+otelcol.receiver.otlp "default" {
+	grpc { }
+
+	http { }
+
+	output {
+		metrics = [otelcol.processor.memory_limiter.default.input]
+		logs    = [otelcol.processor.memory_limiter.default.input]
+		traces  = [otelcol.processor.memory_limiter.default.input]
+	}
+}
+
+otelcol.processor.memory_limiter "default" {
+	check_interval   = "1s"
+	limit_percentage = 90
+
+	output {
+		metrics = [otelcol.exporter.otlp.default.input]
+		logs    = [otelcol.exporter.otlp.default.input]
+		traces  = [otelcol.exporter.otlp.default.input]
+	}
+}
+
+otelcol.exporter.otlp "default" {
+	client {
+		endpoint = "database:4317"
+	}
+}
+```
+
+## Limitations
+
+Configuration conversion is done on a best-effort basis. {{< param "PRODUCT_ROOT_NAME" >}} will issue warnings or errors where the conversion can't be performed.
+
+After the configuration is converted, review the {{< param "PRODUCT_NAME" >}} configuration file created and verify that it's correct before starting to use it in a production environment.
+
+The following list is specific to the convert command and not {{< param "PRODUCT_NAME" >}}:
+
+* Many OpenTelemetry Collector components are supported. You can get a general idea of which exist in Flow mode for conversion by reviewing
+  the `otelcol.*` components in the [Component Reference]({{< relref "../../reference/components/" >}}). Any additional unsupported features are returned as errors during conversion.
+* Check if you are using any extra command line arguments with OpenTelemetry Collector that aren't present in your configuration file.
+* Metamonitoring metrics exposed by {{< param "PRODUCT_NAME" >}} usually match OpenTelemetry Collector metamonitoring metrics but will use a different name.
+  Make sure that you use the new metric names, for example, in your alerts and dashboards queries.
+* The logs produced by {{< param "PRODUCT_NAME" >}} differ from those produced by OpenTelemetry Collector.
+* {{< param "PRODUCT_ROOT_NAME" >}} exposes the {{< param "PRODUCT_NAME" >}} [UI][].
+
+[OpenTelemetry Collector]: https://opentelemetry.io/docs/collector/configuration/
+[debugging]: #debugging
+[example]: #example
+
+{{% docs/reference %}}
```

Comment:
These doc references can stay as-is.. or be converted to relative URIs.

For example: 
```
[Components]: "/docs/agent/ -> /docs/agent/<AGENT_VERSION>/flow/concepts/components.md"
[Components]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/send-data/agent/flow/concepts/components.md"
```

would instead be:
```
[Components]: ../../../concepts/components/
```
(we would have to verify the directory depth)